test/test_mp_all_gather.py

```diff
@@ -8,21 +8,41 @@
 
 def _mp_fn(index):
   device = xm.xla_device()
+  world_size = xm.xrt_world_size()
   if xm.xla_device_hw(device) in ('TPU', 'GPU'):
-    world_size = xm.xrt_world_size()
+    # Testing with a single replica group
     ordinal_tensor = torch.tensor([index], dtype=torch.float).to(device)
-    result = xm.all_gather(ordinal_tensor)
+    result = xm.all_gather(ordinal_tensor, dim=0)
```

Comment:
why do we add `dim=0` here, is it the default?

Reply:
It's default, but to be explicit.